plugins/AuditLog/AuditLog.php

```diff
@@ -557,6 +557,11 @@ public function beforeActivate()
         {
             if (!$this->api->tableExists($this, 'log'))
             {
+                $options = '';
+                if (Yii::app()->db->driverName == 'mysqli' || Yii::app()->db->driverName == 'mysql') {
```

Comment:
I think more of something in API ;) : api->createTable function (but this api is … so … so … light)